docs/src/operator/usage.md

```diff
@@ -1,2 +1,119 @@
+# Configuration and Usage
 
-TBD: waiting on CLI rework to be completed.
+As outlined in the [Architecture](./architecture.md) chapter, the node consists of several components which can be run
+separately or as a single bundled process. At present, the recommended way to operate a node is in bundled mode and is
+what this guide will focus on. Operating the components separately is very similar and should be relatively
+straight-foward to derive from these instructions.
+
+This guide focusses on basic usage. To discover more advanced options we recommend exploring the various help menus
+which can be accessed by appending `--help` to any of the commands.
+
+## Bootstrapping
+
+The first step in starting a new Miden network is to initialize the genesis block data. This is a once-off operation.
+
+```sh
+# Write the default genesis configuration to a file.
+#
+# You can customize this file to add or remove accounts from the genesis block.
+# By default this includes a single public faucet account.
+#
+# This can be skipped if using the default configuration.
+miden-node store dump-genesis > genesis.toml
+
+# Create a folder to store the node's data.
+mkdir data 
+
+# Create a folder to store the genesis block's account secrets and data.
+#
+# These can be used to access the accounts afterwards. Without these the accounts would be inaccessible.
+mkdir accounts
+
+# Bootstrap the node.
+#
+# This generates the genesis block data and stores it in `<data-directory>/genesis.dat`. This is used by the node to
+# create and verify the database during node startup.
+#
+# Account secrets are stored as `<accounts-directory>/account_xx.mac` where `xx` is the index of the account in the
+# configuration file. These account files are not used by the node and should instead be used whereever you intend to operate these accounts,
+# e.g. to run the `miden-faucet` (see Faucet section).
```

Comment:
I would break comment lines here so that they fit onto the page without causing horizontal scroll bars.